### DISCUSSION TITLE
How to create a new admin account in front end Postal server

### DISCUSSION THREAD

**Discussion by mea14** [postalserver/postal#2039] [Help with using Postal]
Hi All,

I require a very urgent help on this topic, i am logging on to the postal server for which i have received the creds as an admin  - "abcde" without "@". When i try to login to console page it says that you are missing this "@". Can someone help in confirming what should be the domain or how it should be logged in by using just username

![MicrosoftTeams-image](https://user-images.githubusercontent.com/17716979/175367991-ac2276cb-8d49-4d76-aadd-bceb9ac094ba.png)

**Comment by willpower232** [comment]
You can only log in with an email address and there has to be one when the user was made so you'll have to consult whoever set up the user to confirm which email address they used.
